src/autobind/autobind.ts

```diff
@@ -12,6 +12,7 @@ const EDITABLE_ELEMENTS_SELECTOR = [
   'input[type=text]',
   '[contenteditable="true"]',
   '[contenteditable="plaintext-only"]',
+  '[contenteditable]',
```

Comment:
According to my understanding,  "[contenteditable]" will also match contenteditable="false". I guess we don't want to  match fields that can't be edited.

Reply:
@acromarco I have replaced '[contenteditable]' with '[contenteditable=""]'. '[contenteditable=""]' is work as '[contenteditable="true"]'.